app/assets/javascripts/osem-switch.js

```diff
@@ -11,4 +11,27 @@ $(function () {
       dataType: 'script'
     });
   });
+
+  $("[class='switch-checkbox-schedule']").bootstrapSwitch();
+
+  $('input[class="switch-checkbox-schedule"]').on('switchChange.bootstrapSwitch', function(event, state) {
+    var url = $(this).attr('url');
+    var method = $(this).attr('method');
+
+    if(state){
+      url += $(this).attr('value');
+    }
+
+    var callback = function(data) {
+      if(data.status != 'ok'){
+        alert("The schedule couldn't been updated");
```

Comment:
I was not sure of that. Maybe it is not even necessary to show a message as the event will go back to its original position if it couldn't be scheduled.